platform/android/MapboxGLAndroidSDKTestApp/build.gradle

```diff
@@ -45,11 +45,9 @@ android {
     buildTypes {
         debug {
             testCoverageEnabled = true
-            minifyEnabled false
```

Comment:
default behaviour is false so it's enforced by default, just trying to limit amount of code here